### ISSUE TITLE
Issues with shared files

### ISSUE THREAD

**Issue by patrickhoehn** [deployphp/deployer#2377]
- Deployer version: v7 beta 13

I copied over my shared files and dirs settings from a working pre v7 beta project but shared dirs are just created but empty and shared files are created with the correct file name but are just empty files.

**Comment by antonmedv** [comment]
What content of such files should be? And what in shared dirs?

**Comment by patrickhoehn** [comment]
In this case the file is a php file, which is created but empty (0 kb). The dirs should contain various sub dirs and files but on the server then, there is only the folder structure defined in the code below but no sub dirs or files in it. In v6 I started my project in development without any shared dirs/files. In production I added those lines and after the next deployment all relevant files were copied to the shared dir. 
```
add('shared_files', [
    'config/statamic/protect.php'
]);
add('shared_dirs', [
    'content',
    'public/assets',
    'users'
]);
```
This time my workaround was uploading the files manually to the shared dirs
